src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div className="dashboard">
+        <div className="container">
+          <div className="two-columns flex jc-center">
+            <div className="column flex fd-column">
+              <div className="block">
+                <Link to="/"><div className="link-block left-top"><h2>Dashboard</h2></div></Link>
```

Comment:
don't like 3-levels deep JSX. Break it down into multiple strings, format it correctly. You can't get any benefits from saving of 2 strings of vertical space if JSX is not readable